src/Control/Optics/Linear/Internal.hs

```diff
@@ -6,14 +6,42 @@
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE TypeOperators #-}
 
-module Control.Optics.Linear.Internal where
+module Control.Optics.Linear.Internal
+  ( -- * Types
+    Optic_(..)
+  , Optic
+  , Iso, Iso'
+  , Lens, Lens'
+  , Prism, Prism'
+  , Traversal, Traversal'
+    -- * Composing optics
+  , (.>)
+    -- * Common optics
+  , swap, assoc
+  , _1, _2
+  , _Left, _Right
+  , _Just, _Nothing
+  , traversed
+    -- * Using optics
+  , get, set, gets
+  , match, match', build
+  , over, over'
+  , traverseOf, traverseOf'
+  , lengthOf
+  , withIso
+    -- * Constructing optics
+  , iso, prism
+  )
+  where
 
 import qualified Data.Bifunctor.Linear as Bifunctor
 import Data.Bifunctor.Linear (SymmetricMonoidal)
 import Data.Functor.Const
 import Data.Functor.Linear
 import Data.Monoid
 import Data.Profunctor.Linear
+import qualified Data.Profunctor.Kleisli.Linear as L
+import qualified Data.Profunctor.Kleisli.NonLinear as NL
```

Comment:
There is no good name for this one. `Hask` could be? `Unrestricted` would be a little confusing… I guess `NonLinear` is clear enough in this context though.